eks-worker-al2.json

```diff
@@ -4,7 +4,7 @@
     "ami_name": "amazon-eks-node-{{timestamp}}",
     "binary_bucket_name": "amazon-eks",
     "binary_bucket_region": "us-west-2",
-    "binary_bucket_path": "1.11.5/2018-12-06/bin/linux/amd64",
+    "binary_bucket_path": "1.12.6/<fill>/bin/linux/amd64",
```

Comment:
It doesn't seem like there's actually a reasonable default here. Why not just remove the default and force the Makefile to set it.

Reply:
Good point. Fixed it.